scripts/upload_peers.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -eou pipefail
+
+LOG_START='\n\e[1;36m'           # new line + bold + color
+LOG_END='\n\e[0m'                # new line + reset color
+DONE_START='\n\e[1;32m'          # new line + bold + green
+DONE_END='\n\n\e[0m'             # new line + reset
+LOG_WARNING_START='\n\e\033[33m' # new line + bold + warning color
+LOG_WARNING_END='\n\e\033[0m'    # new line + reset
+
+SOURCE_PROJECT_ID=keep-test-f3e0
+DESTINATION_BUCKET_NAME="diagnostics_test"
+PROJECT_NAME="keep-test"
+PEERS_LOCATION="diagnostics/peers.json"
+
+help() {
+  echo -e "\nUsage: $0" \
+    "--key-file-path <GCP-key-file-path>"
+  echo -e "\n\nCommand line arguments:\n"
+  echo -e "\t--key-file: GCP key file path\n"
+  exit 1 # Exit script after printing help
+}
+
+# Transform long options to short ones
+for arg in "$@"; do
+  shift
+  case "$arg" in
+  "--key-file") set -- "$@" "-k" ;;
+  "--help") set -- "$@" "-h" ;;
+  *) set -- "$@" "$arg" ;;
+  esac
+done
+
+# Parse short options
+OPTIND=1
+while getopts "k:h" opt; do
+  case "$opt" in
+  k) key_file_path="$OPTARG" ;;
+  h) help ;;
+  ?) help ;; # Print help in case parameter is non-existent
+  esac
+done
+shift $(expr $OPTIND - 1) # remove options from positional parameters
+
+KEY_FILE_PATH=${key_file_path:-""}
+
+if [ "$KEY_FILE_PATH" == "" ]; then
+  printf "${LOG_WARNING_START}Key file must be provided.${LOG_WARNING_END}"
+fi
+
+# Run script
+printf "${LOG_START}Authenticating with gcloud...${LOG_END}"
+gcloud auth activate-service-account --key-file $key_file_path
+
+printf "${LOG_START}Setting cloud context to the ${PROJECT_NAME}...${LOG_END}"
+gcloud config set project $SOURCE_PROJECT_ID
+
+printf "${LOG_START}Uploading peers info to a ${DESTINATION_BUCKET_NAME} bucket...${LOG_END}"
+gcloud storage cp ${PEERS_LOCATION} gs://$DESTINATION_BUCKET_NAME/
```

Comment:
I like using `gcloud` but I'm afraid it might cause some problems on the bootstrap VM, as this is an additional tool that has to be installed and maintained. 

Could we try a simple way, e.g. with `curl`? 

Resources:
- https://cloud.google.com/storage/docs/uploading-objects#rest-upload-objects
- https://stackoverflow.com/questions/42418493/how-do-i-upload-to-google-cloud-storage-with-curl-and-an-api-key

Reply:
This was also my concern but since we already used `gcloud` in our scripts, I figured we can use it here as well. However, I agree that it might be an additional tool that a bootstrap node has to keep an eye on, so here's a change [d154f99](https://github.com/keep-network/keep-core/pull/3255/commits/d154f996b3090424477429f5f1d10fe2b64a417c)